packages/babel-types/src/definitions/es2015.js

```diff
@@ -120,7 +122,9 @@ defineType("ExportDefaultDeclaration", {
   visitor: ["declaration"],
   aliases: ["Statement", "Declaration", "ModuleDeclaration", "ExportDeclaration"],
   fields: {
-    // todo
+    declaration: {
+      validate: assertNodeType("Declaration", "Expression")
```

Comment:
Although when using `Declaration` - you aren't allowed to have a `VariableDeclaration` here so maybe we should put `FunctionDeclaration` and `ClassDeclaration` instead